db/migrate/20191210090306_add_user_id_to_places.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToPlaces < ActiveRecord::Migration[5.2]
+  def change
+    add_column :places, :user_id, :integer
```

Comment:
This is wrong migration we talk some time ago that you need also index for this column. Please check the documentation and fix that.

Reply:
Ok... I thought that in rails 5 index is automatically created. 

So is it a right direction now:
rails generate migration AddIndexToUserIds
and then: 
```
def change
    add_index :places, :user_id
  end
```

Reply:
You can do it manually, but please check how did we create this kind of field in the past. It was created by default with index but you need to use different method for that.

Reply:
AddIndexToPlaceUserRelation ?

Reply:
This is true that in some cases name of migration can help you with preparing code inside migration, but when you will not specify the right type of the field index will be not created.